OpenSim/Simulation/Manager/Manager.h

```diff
@@ -130,19 +130,13 @@ class OSIMSIMULATION_API Manager
 // METHODS
 //=============================================================================
 public:
-    /** This constructor cannot be used in MATLAB/Python, since the
-     * SimTK::Integrator%s are not exposed in those languages. */
-    Manager(Model& model, SimTK::Integrator& integ);
     /** Constructor that takes a model only and internally uses a
      * SimTK::RungeKuttaMersonIntegrator with default settings (accuracy,
      * constraint tolerance, etc.). MATLAB/Python users must use this
      * constructor. */
```

Comment:
"MATLAB/Python users must use this constructor" is no longer true; the other constructor can also be used from scripting.